app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+  {{super()}}
+  <script type="module" src="/static/js/manageServiceLearningFaculty.js"></script>
+  <script type="text/javascript" src="https://cdn.datatables.net/v/dt/dt-1.11.3/r-2.2.9/datatables.min.js"></script>
+{% endblock %}
+
+{% block app_content %}
+
+<div id="wrapper">
+  <div class ="container">
+    <a class="skipContent" href="#backButton" tabindex="1" style="top:8%">Click to Skip</a>
+    <div class="contributors" align="center">
+      <h2> Manage Service Learning Faculty </h2>
+  <table class="table table-striped" id="myTable">
+    <thead>
+    <tr class="header">
+    <th scope = "col">Name</th>
+    <th scope = "col">Prev Classes</th>
+    </tr>
+    </thead>
+  <tbody id = "facultyTable">

```

Comment:
This is inside the table. It should be the same indentation level as `<thead>`. There are still some other indentation issues to look at.

Reply:
fixed